swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/generator/core/ParameterTypeProcessor.java

```diff
@@ -17,5 +17,5 @@
 package io.servicecomb.swagger.generator.core;
 
 public interface ParameterTypeProcessor {
-  void process(OperationGenerator operationGenerator, int paramIdx);
+  boolean process(OperationGenerator operationGenerator, int paramIdx);
```

Comment:
maybe no need to change this?
in io.servicecomb.swagger.generator.core.OperationGenerator.scanMethodParameters():
after io.servicecomb.swagger.generator.core.OperationGenerator.processByParameterType(Type, int), use swaggerParamCount compare to check if a new parameter already added.

and maybe change to new order is better:
1.process by annotation
2.process by parameter type
3.process by default processor

Reply:
if someone added annotation to HttpServletRequest or InvocationContext, the param processing will be different. are you sure that's okay?